### PR TITLE
Enable support for running Cerberus on Power (ppc64le)

### PR DESCRIPTION
**Pull request by juliemathew**

### CODE REVIEW

File: containers/Dockerfile.ppc64le [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Dockerfile for cerberus for ppc64le arch
+
+FROM quay.io/openshift/origin-tests:latest as origintests
```
[the remark this answers is not in the document]
Comment (juliemathew) [reply containers/Dockerfile.ppc64le]: Done. Please review @chaitanyaenr . Thanks.

File: containers/Dockerfile.ppc64le [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Dockerfile for cerberus for ppc64le arch
+
+FROM ppc64le/centos:8
```
Comment (chaitanyaenr) [inline containers/Dockerfile.ppc64le:3]: @juliemathew This is great. We might want to add document around using an image built using this dockerfile when running Cerberus on Power: https://github.com/cloud-bulldozer/cerberus/blob/master/docs/installation.md#containerized-version. Thoughts?

Comment (juliemathew) [reply containers/Dockerfile.ppc64le:3]: yes, I am working on this @chaitanyaenr

Comment (juliemathew) [reply containers/Dockerfile.ppc64le]: I have added required documentation for supporting Cerberus tool on Power arch. 
Please review @chaitanyaenr  .Thank you.

**Comment by mffiedler** [comment]
Hi @juliemathew Can you squash the commits for final review?  Thanks!

**Comment by juliemathew** [comment]
> 
> 
> Hi @juliemathew Can you squash the commits for final review? Thanks!

Done.  
Could you please take it for final review. Thanks.
@mffiedler

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Pull request merged**